README.md

```diff
@@ -1,20 +1,16 @@
-# Intermediary mappings
+# Intermediary
 
-This repository contains the match information between different versions of Minecraft created by the Fabric project, as well as the "intermediary" mappings we use, that is an intermediary naming form which tries to keep names
-the same across versions and mapping changes.
+Public facing repo for Yarn Intermediary mappings.
 
-## Files included in this repo
+This is NOT intended for end users to use. This IS a replacement for MCP. This WILL allow you to make Mods. PLEASE ASK. This is inteded for making the entire process of updating MC obfusication mappings public. As well ensuring that this process grants freedom to the user so that others may use them how they please.
 
-* __mappings/<mcversion>.tiny__: Intermediary mappings in the Tiny mapping format.
-* __matches/*.matches__: Matches between Minecraft versions, created by the Fabric project with the aid of Matcher.
+You MAY modify/rehost the generated mappings without express consent from either LexManos or Searge. This is NOT intended to prevent conflicting mappings from being published as that does nothing but monopolize the mappings process. You MAY download our mappings from Fabric's maven, and use them in your project. You MAY give proper credit in some way but are not REQUIRED to do so.
```

Comment:
`You MAY modify/rehost the generated mappings without express consent from either LexManos or Searge.` this has nothing to do with this repo? im not sure what you are trying to do here.